docs/integrations/index.mdx

```diff
@@ -1,9 +1,10 @@
 ---
+pagination_next: integrations/aftership/index
```

Comment:
I'm wondering if we disable for this, since this page is fully generated.

Reply:
Yeah, this was the biggest issue one. I can make it `null`